### DISCUSSION TITLE
Cisco Catalyst Power Supplys, Stack calbes

### DISCUSSION THREAD

**Discussion by kiksen** [netbox-community/devicetype-library#1526] [Q&A] [answered]
Hello,
I have two questions.
I saw that there was an discussion about modeling power modules. Are there rules how to name power supply module bays? Like PS-A and PS-B?
StackWise Links are done as interfaces, not modules right?
Thank you!

**Comment by DanSheps** [comment, marked as answer]
Stackwise links are done as interfaces, however there is a module for a specific version of the 9000 series for stackwise (it should also include stack interfaces)

I prefer PSU-A however we might want to revisit that and use the "official" convention from `show inventory` which is "Power Supply A", "Power Supply B", for Catalysts (where it is appropriate).
